content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -32,6 +40,8 @@ To learn how Istio handles tracing, visit this task's [overview](../overview/).
 1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage` one or more times
     to generate trace information.
 
+    {{< boilerplate trace-generation >}}
```

Comment:
Please replace with the numbered steps the readers need to take.

Reply:
There aren't any numbered steps needed to be taken. This text is merely a further explanation of the first step.